proto/decentraland/kernel/apis/communications_controller.proto

```diff
@@ -8,7 +8,13 @@ message RealSendRequest {
 message RealSendResponse {}
 
 message SendBinaryRequest {
+  repeated bytes data = 1; // broadcasted messages
+  repeated PeerMessageData peer_data = 2; // peer-to-peer messages
+}
+
+message PeerMessageData {
   repeated bytes data = 1;
+  string address = 2;
```

Comment:
```suggestion
  repeated string addresses = 2;
```

I think this can be an array, LiveKit supports it
https://github.com/livekit/protocol/blob/4aff238b1235618939dd10dd5aae21a776ad1b3c/protobufs/livekit_room.proto#L186C1-L197C2